src/components/currentFilm/currentFilmDescription.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './currentFilmDescription.css';
+
+export default function CurrentFilmDescription () {
+    return (
+        <header>
+           <div className="currentFilmContainer">
+            <img className="currentFilmContainer_poster" src="https://image.tmdb.org/t/p/w600_and_h900_bestv2/sWgBv7LV2PRoQgkxwlibdGXKz1S.jpg" alt="what's wrong"/>
+            <div className="currentFilmContainer_detailsContainer">
+                <div className="currentFilmContainer_detailsContainer-titleContainer">
```

Comment:
Why currentFilm in header tag? It has to be a single component

Reply:
If you like using header and footer tags please split your component into logical blocks.